holmes/plugins/toolsets/infrainsights/redis_toolset.py

```
@@ -0,0 +1,534 @@
+    
+    def get_client(self) -> redis.Redis:
+        """Get the Redis client"""
+        if not self.client:
+            self._connect()
+        return self.client
+
+
+class GetRedisInfo(BaseInfraInsightsTool):
+    """Get Redis server information and statistics"""
+    
+    def __init__(self, toolset: "RedisToolset"):
```

Ruff (0.12.2)

Undefined name `RedisToolset`

(F821)